src/Utils/RandomIntegerInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\BannerServer\Utils;
```

Comment:
The interface belongs to the `Entity` namespace: The domain classes should not have "outside" dependencies.